device_doctor/bin/main.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/args.dart';
+
+import 'package:device_doctor/src/config.dart';
```

Comment:
Prefer relative imports - https://dart-lang.github.io/linter/lints/prefer_relative_imports.html

Reply:
That seems for files located within lib?